embedded-graphics/src/primitives/rounded_rectangle.rs

```diff
@@ -162,6 +162,48 @@ impl RoundedRectangle {
         Self::new(rectangle, CornerRadii::new(corner_radius))
     }
 
+    /// Return the rounded rectangle with confined corner radii.
```

Comment:
I would add a note that calling this method isn't necessary most of the time, because all operations provided by e-g confine the radii internally.